src/main/resources/application.yml

```diff
@@ -86,6 +86,10 @@ management:
     info:
       cache:
         time-to-live: 2000ms
+  metrics:
+    distribution:
+      percentiles:
+        http.server.requests: 0.9,0.99
```

Comment:
Do we get a mean/median time by default? I think whilst the 99% centile is generally better for limits, understanding what a middle of the road consumer gets can be very ehlpful

Reply:
Why is prometheus so much better than cloudwatch? https://micrometer.io/docs/concepts#_histograms_and_percentiles

Reply:
We get mean by default